### PR TITLE
Grafana builder: use $__rate_interval over $__interval

### PR DESCRIPTION
**Pull request by bergquist**
Signed-off-by: bergquist <carl.bergquist@gmail.com>

### CODE REVIEW

File: grafana-builder/grafana.libsonnet [hunk 1]
```diff
@@ -399,23 +399,23 @@
     nullPointMode: 'null as zero',
     targets: [
       {
-        expr: 'histogram_quantile(0.99, sum(rate(%s_bucket%s[$__interval])) by (le)) * %s' % [metricName, selector, multiplier],
+        expr: 'histogram_quantile(0.99, sum(rate(%s_bucket%s[$__rate_interval])) by (le)) * %s' % [metricName, selector, multiplier],
```
Comment (bergquist) [inline grafana-builder/grafana.libsonnet:402]: I've never used `histogram_quantile` so I dont know if this makes sense.. But I dont see why It wouldn't :man_shrugging:

**Comment by tomwilkie** [comment]
When was rate_interval introduced?  This will break people on old Grafana IIUC.  Just want to make sure old in this context doesn't mean "last week".

**Comment by bergquist** [comment]
It was introduced in Grafana 7.2.0 released (2020-09-23)

ref https://github.com/grafana/grafana/pull/26937

**Comment by tomwilkie** [comment]
Awesome, thanks Carl.  I'll leave it to you to judge if 3 months is enough - this library is used by the k8s mixin, which is included in the kube-prometheus, which embeds grafana/grafana:7.3.5 in head (see https://github.com/prometheus-operator/kube-prometheus/blob/master/manifests/grafana-deployment.yaml).  So I'd say LGTM

**Comment by bergquist** [comment]
3 months in mixin lands is how many human years?

**Pull request merged**